main.go

```diff
@@ -199,12 +205,16 @@ func main() {
 	//
 	// Server
 	//
-
 	httpTimeout := 10 * time.Minute
 	maxHeaderBytes := 10 * (1024 * 1024)
 
+	if tlsCertificate != "" && tlsKey != "" {
```

Comment:
This is a clean way of making sure we don't implement a breaking change. Nice work!